src/plugins/xerces/README.md

```diff
@@ -16,36 +16,36 @@ formatted files. It uses a general format which:
 - Maps key names to XML elements
 - Maps key values to textual content of XML elements
 - Maps metakeys to XML attributes. Metakey name = attribute name, Metakey value
- 	= attribute value
+  = attribute value
 - Ignores XML comments
 
 ## Usage
 
 To mount an XML file we use:
 
-	kdb mount file.xml user/test/file xerces
+    kdb mount file.xml user/test/file xerces
```

Comment:
Is it possible to use tabs instead of spaces?

Reply:
There is an option to use tabs instead of spaces for indentation. However, even if this option is set, `prettier` will still use spaces to indent code sections:

```sh
# The following command produces 4 space characters followed by the text `code`
printf '\tcode' | prettier --stdin --stdin-filepath test.md --use-tabs
#>    code
```

.

Reply:
Ok, then better consistently use spaces.